examples/gatsbygram/package.json

```diff
@@ -38,10 +38,16 @@
   "main": "n/a",
   "scripts": {
     "lint": "./node_modules/.bin/eslint --ext .js,.jsx --ignore-pattern public .",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "start-server-and-test develop http://localhost:8000 cy:run",
     "develop": "gatsby develop",
     "dev": "gatsby develop",
     "build": "gatsby build",
-    "deploy": "gatsby build --prefix-paths && gh-pages -d public"
+    "deploy": "gatsby build --prefix-paths && gh-pages -d public",
+    "cy:open": "cypress open",
+    "cy:run": "cypress run --record"
```

Comment:
@tsriram If this `--record` key is removed, that should make the CI tests pass again? 

Then `--record` can be added back in a follow-up PR, along with the env var.

Reply:
Yep, makes sense. Will remove `--record` for now :+1: